tssc/step_implementers/package/maven.py

```diff
@@ -12,7 +11,55 @@
 
 .. Important::
 
-    If package not specfied in pom will default to jar in result.
+    If package not specified in pom will default to jar in result.
+
+Step Configuration
+------------------
+
+Step configuration expected as input to this step.
+Could come from either configuration file or
+from runtime configuration.
+
+| Configuration Key     | Required? | Default                 | Description
+|-----------------------|-----------|-------------------------|-----------
+| `pom-file`            | True      | `'pom.xml'`             | Maven pom file to build
+| `artifact-extensions` | True      | `["jar", "war", "ear"]` | Extensions to look for in the
+                                                                `artifact-parent-dir` for built
+                                                                artifacts.
+| `artifact-parent-dir` | True      | `'target'`              | Parent directory to look for built
+                                                                artifacts in ending in
+                                                                `artifact-extensions`.
+
+Expected Previous Step Results
+------------------------------
+
+Results expected from previous steps that this step requires.
+
+None.
+
+Results
+-------
+
+Results output by this step.
+
+| Result Key  | Description
+|-------------|------------
+| `artifacts` | An array of dictionaries with information on the built artifacts.
+
+
+**artifacts**
+Keys in the dictionary elements in the `artifacts` array in the step results.
+
+| `artifacts`Key | Description
```

Comment:
@itewk Minor space issue -> 'artifacts'_Key

Reply:
thanks. fixing in new commit.